Fix type completor section of README (#820)
ruby/irb: README.md

### How to Enable IRB::TypeCompletor

Install [ruby/repl_type_completor](https://github.com/ruby/repl_type_completor/) with:
```
$ gem install repl_type_completor
```
Or add these lines to your project's Gemfile.
```ruby
gem 'irb'

> gem 'repl_type_completor', group: :development

Reviewer: Maybe `:test` group too? Otherwise users won't get the same completion behaviour when adding `binding.irb` in tests?

```

Now you can use type based completion by:

> Run IRB with `--type-completor` option

Reviewer: ```suggestion
Running IRB with the `--type-completor` option
```

```
$ irb --type-completor
```

> Or write this line to IRB's rc-file

Reviewer: ```suggestion
Or writing this line to IRB's rc-file (e.g. `~/.irbrc`)
```

```ruby
IRB.conf[:COMPLETOR] = :type # default is :regexp
```

> Or use environment variable `IRB_COMPLETOR`

Reviewer: ```suggestion
Or setting the environment variable `IRB_COMPLETOR`
```
